odoo/addons/base/models/ir_mail_server.py

```diff
@@ -530,7 +541,9 @@ def _prepare_email_message(self, message, smtp_session):
         smtp_to_list = [
             address
             for base in [email_to, email_cc, email_bcc]
-            for address in extract_rfc2822_addresses(base)
+            # be sure a given address does not return duplicates (but duplicates
+            # in final smtp to list is still ok)
+            for address in extract_rfc2822_addresses_unique(base)
```

Comment:
You could have just used:
```suggestion
            for address in tools.unique(extract_rfc2822_addresses(base))
```